atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/http/TimelockUtils.java

```diff
@@ -43,21 +46,26 @@ private TimelockUtils() {
     }
 
     private static List<String> hostnamesToEndpointUris(List<String> hosts) {
-        return Lists.transform(hosts, host -> String.format("https://%s:%d/%s", host, PORT, NAMESPACE));
+        return hosts.stream().map(host -> String.format("https://%s:%d", host, PORT)).collect(Collectors.toList());
     }
 
     private static <T> T createFromUris(MetricsManager metricsManager, List<String> endpointUris, Class<T> type) {
+        ServerListConfig serverListConfig = ImmutableServerListConfig.builder()
+                .addAllServers(endpointUris)
+                .sslConfiguration(SSL_CONFIGURATION)
+                .build();
+        // lock technically blocking, but OK to run with short timeout given the way the tests work.
```

Comment:
why is this okay? what happens if you make it the same as how it's used in prod?

Reply:
Is also fine, just that I'm not really a fan of duplicating that logic here, and it seems a bit unpleasant to expose internals of `TransactionManagers` to simulate that wrapping. Let me know what you think.

Reply:
I suppose I'm asking from a perspective of "will jepsen say there's an illegal trace, when really it would never happen in prod"?